src/dotnet-aspnet-codegenerator/ToolCommandLineHelper.cs

```diff
@@ -17,6 +17,7 @@ internal class ToolCommandLineHelper
         private const string PROJECT = "--project";
         private const string NUGET_PACKAGE_DIR = "--nuget-package-dir";
         private const string BUILD_BASE_PATH = "--build-base-path";
+        private const string DISPATCHER_VERSION = "--dispatcher-version";
```

Comment:
Hmm, if you need this might be worth making a PR to common to expose it so it's not hardcoded here.